.env.development

```diff
@@ -0,0 +1,13 @@
+# Set docker-compose ENV overrides below:
+
+# ------------------------
+# Hokusai Dev Environment
+#
+# e.g. When running both MP and Gravity in docker via `hokusai dev start`
+# ------------------------
+GRAVITY_API_BASE=http://gravity:3000/api/v1
```

Comment:
This assumes that gravity middleware has been [configured](https://guides.rubyonrails.org/v6.1/configuring.html#configuring-middleware) to permit connections for `gravity` host. As is gravity will block that host.